src/routes/command-type/add.ts

```diff
@@ -26,7 +26,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithSupplier = {
   method: 'post',
   path: '/',
   body: (apiVersion: number): Joi.ObjectSchema => Joi.object().keys({
-    name: stringOrTranslationsSchema.required().example('Measurement cycle'),
+    name: stringOrTranslationsSchema().required().example('Measurement cycle'),
```

Comment:
If I read the code correctly this currently gives no limit for the name in the request. I thought limiting this in the request was the main reason we are even changing the sdk? Currenly you seem to mainly limit stuff in the models but then it does not fail as fast as it could?

Reply:
Indeed

Reply:
1cd34ee